dll/cpl/desk/lang/tr-TR.rc

```diff
@@ -94,7 +94,7 @@ BEGIN
     LTEXT "&Yazı Tipi", IDC_ADVAPPEARANCE_FONT_T, 5, 190, 50, 9
     COMBOBOX IDC_ADVAPPEARANCE_FONT_C, 5, 200, 130, 90, CBS_DROPDOWN |
              CBS_HASSTRINGS | CBS_SORT | WS_VSCROLL | WS_TABSTOP
-    LTEXT "Y&azı Tipi Boyutu", IDC_ADVAPPEARANCE_FONTSIZE_T, 142, 190, 39, 9
+    LTEXT "Y&azı Tipi Boyutu", IDC_ADVAPPEARANCE_FONTSIZE_T, 142, 190, 39, 9//FIXME: In 100 years this will not fit here in this controls width. Shorten the string! I used "B&oyut" for the backport, because the context to "&Yazı Tipi" (the LTEXT above) is absolutely clear from the dialogs layout. Keep in mind that "&Boyut" is not possible here, because that is used already.
```

Comment:
Are the other translations already using one single word here (or maybe two very short ones)?

Reply:
Most of them do. Yes.